eng/build.yml

```diff
@@ -72,20 +72,7 @@ phases:
           PublishType: $(_PublishType)
           TestFullMSBuild: 'true'
 
-    - ${{ if eq(parameters.agentOs, 'Windows_NT_TestAsTools') }}:
```

Comment:
We should be able to leave this one around, since it won't be invoked due to the bigger YAML not having an agentOS for it. But your choice really.

Reply:
I see, I just reverted the files to before the commits that changed them. It can be re-enabled with a revert of this commit, so I don't think it matters.